coalib/parsing/FilterHelper.py

```diff
@@ -33,3 +33,20 @@ def get_filtered_bears(cls, filter, args, all_bears=None):
         if not args or len(args) == 0:
             return all_bears
         return cls.available_filters[filter](all_bears, args)
+
+    @classmethod
+    def filter_bears(cls, bears, args):
```

Comment:
Hmm `apply_filter` & `apply_filters` sounds good. Should I add another commit for rename ?